src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/RazorLanguageEndpointTest.cs

```diff
@@ -95,7 +95,7 @@ public async Task Handle_MapToDocumentRanges_CSharp_Unmapped()
             var response = await Task.Run(() => languageEndpoint.Handle(request, default));
 
             // Assert
-            Assert.Equal(RazorLanguageEndpoint.UndefinedRange, response.Ranges[0]);
+            Assert.Equal(RangeExtensions.UndefinedRange, response.Ranges[0]);
```

Comment:
Same question about object.referenceEquals on all of these.

Reply:
https://github.com/dotnet/aspnetcore-tooling/pull/2832#discussion_r544704967